cdhweb/templates_src/base.html

```diff
@@ -0,0 +1,63 @@
+{% load static wagtailuserbar wagtailcore_tags wagtailimages_tags %}
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}{% block title_suffix %}{% endblock %}</title>
+    <meta name="description" content="{% firstof self.search_description self.intro|default:''|striptags self.body_excerpt|default:''|striptags|truncatewords:40 self.title %}">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <script type="text/javascript"> window.staticRoot = "{% static 'dist/' %}"; </script>
+
+    {% comment %}
+    {% include "core/includes/icons.html" %}
+    {% include "core/includes/meta/twitter_card.html" %}
+    {% include "core/includes/meta/open_graph.html" %}
+
+    {% block og_images %}
+      {% include "core/includes/meta/open_graph_image.html" %}
+    {% endblock %}
+
+    {% endcomment %}
+    {% block extra_css %}
+      {# Needed for specific cases #}
+    {% endblock %}
+
+  </head>
+  <body class="{% block body_class %}{% endblock %}">
+    
+    {% block content %}{% endblock %}
+    {% block main %}{% endblock %}
```

Comment:
Not sure whether both are needed

Reply:
Yeah, seems an arbitrary choice. I'd certainly be inclined to pick just one, for consistency